R/glmCommonFunctions.R

```diff
@@ -55,34 +55,70 @@
   # make formulas for the full model and the null model
   ff <- .createGLMFormula(options, nullModel = FALSE)
   nf <- .createGLMFormula(options, nullModel = TRUE)
```

Comment:
So if we do it like this, it should work without the workaround:
```suggestion
  # make sure that the formula get the current environment (https://github.com/jasp-stats/jaspRegression/pull/179#discussion_r1017412764)
  ff <- .createGLMFormula(options, nullModel = FALSE)
  environment(ff) <- environment()
  nf <- .createGLMFormula(options, nullModel = TRUE)
  environment(nf) <- environment()
  
  weights <- dataset[[options[["weights"]]]]
```

Reply:
thanks. changed it.